src/services/storage/handlers/offer.ts

```diff
@@ -95,7 +95,7 @@ const handlers: { [key: string]: Function } = {
     }
   },
   async BillingPlanSet ({ returnValues: { period, price, token } }: EventData, offer: Offer, offerService: OfferService): Promise<void> {
-    await updatePrices(offer, period, price, token)
+    await updatePrices(offer, period, price, token.toLowerCase())
```

Comment:
hmm.. we have an inconsistency in the cache and contract then, should we also lower case token address when creating a transaction to the storage contract?

Reply:
No because we should allow users to use the contract with either format.